ui/app/ot/controller/cancelSurgicalBlockController.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+angular.module('bahmni.ot').controller('cancelSurgicalBlockController', [
```

Comment:
This is controller for cancel popup. Should we rename it?( cancelSurgicalBlockPopUpController)
@mks-d

Reply:
@ShruthiPitta absolutely.